pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -125,6 +131,12 @@ type GCECloud struct {
 	useMetadataServer        bool
 	operationPollRateLimiter flowcontrol.RateLimiter
 	manager                  diskServiceManager
+	// nodeZones is a mapping from Zone to a sets.String of Node's names in the Zone
+	// it is updated by the nodeInformer
+	nodeZones          map[string]sets.String
+	nodeInformerSynced cache.InformerSynced
+	// Lock to access to node informers
+	nodeZonesLock sync.Mutex
```

Comment:
nit: can you please move it before "nodeZones" - we generally declare the lock before vars it is protecting.
Also, please update its comment.